portfolio/src/main/webapp/woof.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+<body>
+  <h1> Woof. </h1>
+  <!-- TODO: Move height to CSS -->
+  <img src="/images/pet/IMG-0057.JPG" alt="Beef jerky loving dog" height="500">
+  <p> Meet Dukgu, traditional Korean Jindo dog. He just turned 2 years so he is very young.
+      He was very small when he first came, and our family sort of thought he would stay that way for a while, but he grew exponentially every week; still he is super adorable.
+      Jindo dogs are typically hunter dogs so are very smart and loyal, but our dog is different.
+      He doesn't know whether there is a stranger in the house until someone reaches to our front door.
+      He often skips meal because he is scared of mouse near his food tray (He eats right after we remove mouse for him).
+      He also can't play catch because when I throw a ball, he runs to it and fetches it, but instead of giving back he constantly runs around with it in his mouth until he gets tired, which then he just lies down.
+      Still, he was the best addition to our family. 
```

Comment:
nit: the diff shows that you don't have a newline at the end. I think usually we always have a trailing newline character: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file